modules/twinklewarn.js

```diff
@@ -1479,6 +1488,79 @@ Twinkle.warn.callback.postCategoryCleanup = function twinklewarnCallbackPostCate
 	}
 };
 
+// Build the expected edit summary to allow user to make edits through a text box
+Twinkle.warn.callback.set_tentative_edit_summary = function twinkleWarnCallbackSetTentativeEditSummary(e) {
+	var selected_main_group = e.target.form.main_group.value;
+	var selected_template = e.target.form.sub_group.value;
+	var messageData = $(e.target.form.sub_group).find('option[value="' + $(e.target.form.sub_group).val() + '"]').data('messageData');
```

Comment:
There has to be a less error-prone (what if the value contains special chars?) and more elegant way to do this. 

Maybe something like `$(e.target.form.sub_group).find('option:selected')` ?